controller/clusters.go

```diff
@@ -92,6 +92,61 @@ func (c *ClustersController) ShowAuthClient(ctx *app.ShowAuthClientClustersConte
 	return ctx.OK(&clusters)
 }
 
+// Show returns the single clusters.
+func (c *ClustersController) Show(ctx *app.ShowClustersContext) error {
+	if !auth.IsSpecificServiceAccount(ctx, auth.OsoProxy, auth.Tenant, auth.JenkinsIdler, auth.JenkinsProxy, auth.Auth) {
+		log.Error(ctx, nil, "unauthorized access to cluster info")
+		return app.JSONErrorResponse(ctx, errors.NewUnauthorizedError("unauthorized access to cluster info"))
+	}
+	clustr, err := c.app.ClusterService().Load(ctx, ctx.ClusterID)
+	if err != nil {
+		return app.JSONErrorResponse(ctx, err)
+	}
+	return ctx.OK(&app.ClusterSingle{
+		Data: &app.ClusterData{
```

Comment:
As we already have this defined `List`. I think we can move it one function and reuse it from there to follow DRY principle

Reply:
here the `app.ClusterData` is filled with data coming from the DB, whereas in the other case, the data comes from the config. The go types are different, so there's not much we can factorize.

Reply:
Let's address this in https://github.com/fabric8-services/fabric8-cluster/issues/46